orchestra/frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
how much of this is necessary?

Reply:
Hmm it's only necessary if we're making it into a desktop/mobile PWA. I think we can remove this for now.

Reply:
OK cool---let's remove!